src/utils/api/utils.ts

```diff
@@ -22,3 +23,9 @@ export function getGitHubGraphQLUrl(hostname: Hostname): URL {
 
   return url;
 }
+
+export function parseNextUrl(response: AxiosResponse): string | null {
```

Comment:
Parse for what? :)

Reply:
Should probably instead be named after what it's parsing out and returning.

Reply:
`getNextURLFromLinkHeader` ?